scripts/package/package-contributors.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs');
```

Comment:
I could be wrong, but I'm working on the assumption that including Wiki contributors is more or less a licensing requirement. Definitely I agree that in the future we should use GitHub for this.

Really, including these .md files was just the quickest way I could think of to ensure that contributor information was present. I agree with your comment above that giving it more structure would be good, but also we should have a more general think about what our obligations are here and how we can best fulfill them.

Reply:
Yes, if we're deriving content from the existing wiki, then we must attribute the authors. But you're right that this is easy and you shouldn't consider this a blocker to merging.

My thinking here was just that we'll want to—at some point before actually inviting the community to edit Stumptown content—take the authorship data out of the general content files. Once we break from the wiki, the contribution data is historical and Git commits are the canonical source of authorship information. We wouldn't want to update authorship data except via Git from then on, but leaving authorship as part of content files might be confusing to contributors.

For wiki authors who used their GitHub account on MDN, we could conceivably [add a commit which makes them contributors to this repo](https://help.github.com/en/articles/creating-a-commit-with-multiple-authors), whether they actually contributed via GitHub or not. This would further shrink the need for attribution data in the repo.

Reply:
> we'll want to...take the authorship data out of the general content files

If we build this content to HTML and start serving it from developer.mozilla.org, I'm not sure what our licensing requirements are. Currently MDN displays all contributors in the page. Is it sufficient to have a link back to GitHub and let people see the contribution history there (plus something for legacy contributors)? I don't know, but will start a conversation about this. 

> For wiki authors who used their GitHub account on MDN, we could conceivably add a commit which makes them contributors to this repo, whether they actually contributed via GitHub or not. This would further shrink the need for attribution data in the repo.

I seem to remember John had some scheme like this.